src/Sylius/Behat/Context/Transform/PaymentMethodContext.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Payment\Repository\PaymentMethodRepositoryInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+final class PaymentMethodContext implements Context
+{
+    /**
+     * @var PaymentMethodRepositoryInterface
+     */
+    private $paymentMethodRepository;
+
+    /**
+     * @param PaymentMethodRepositoryInterface $paymentMethodRepository
+     */
+    public function __construct(PaymentMethodRepositoryInterface $paymentMethodRepository)
+    {
+        $this->paymentMethodRepository = $paymentMethodRepository;
+    }
+
+    /**
+     * @Transform payment method :name
```

Comment:
If we want to make this (comment)[https://github.com/Sylius/Sylius/pull/4742/files#r60367687] applicable we should add another transform definition
```
@Transform :paymentMethod
```